packages/feature-flags/src/flags.ts

```diff
@@ -6,5 +6,4 @@ export const displayContents = false;
 export const adminRole = false;
 // A general flag to enable/disable all the AI features.
 export const ai = process.env.NODE_ENV !== "production";
```

Comment:
Please replace NODE_ENV check with false. In dev we show features by settings FEATURES=* in .env or in localstorage

Reply:
its enabled in preview, we have there asterix as a value that enables all features, recommend setting the same in your .env locally